packages/core/src/App/Containers/Modals/tnc-status-update-modal/tnc-status-update-modal.scss

```diff
@@ -0,0 +1,43 @@
+.dc-modal__container_tnc-status-update-modal-wrapper {
+    padding: 24px;
+    @include mobile-screen {
+        padding: 16px;
+    }
+}
+
+.tnc-status-update-modal {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+
+    &__header {
+        @include mobile-screen {
+            font-size: 1.4rem;
+        }
+    }
+
+    &__text-container {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        align-self: stretch;
+        .dc-text {
+            @include mobile-screen {
+                font-size: 1.2rem;
+            }
+        }
+    }
+
+    &__button {
+        align-self: flex-end;
+    }
+
+    @include mobile-screen {
+        gap: 16px;
+    }
+}
+
+.link {
+    text-decoration: underline;
+    font-weight: normal;
```

Comment:
why not pass it as prop to component?

Reply:
We don't have a prop for this component :(